.circleci/config.yml

```diff
@@ -197,8 +197,7 @@ jobs:
                 command: |
                   source scripts/utils.sh
                   tag=v$(extract_version_from_current_wd)
-                  # Newlines (<br/>) are necessary for better layout on Marketplace, but make the formatting worse on GitHub
-                  change_notes=$(sed '/^<br.>$/d' CHANGE-NOTES.html)
+                  change_notes=$(./gradlew -q getReleaseChangeLogForGithub)
```

Comment:
Pls also take a look at `scripts/enforce-change-notes-updated-on-pr-to-master` &mdash; this script can now probably be simplified/revamped now. Its purpose should be to make sure that a new release gets a non-empty corresponding section in change notes.